dbt/adapters/dremio/connections.py

```diff
@@ -138,6 +143,9 @@ def __post_init__(self):
 
 class DremioConnectionManager(SQLConnectionManager):
     TYPE = "dremio"
+    DEFAULT_RETRIES = 1
+
+    retries = DEFAULT_RETRIES
```

Comment:
Should this be private?

"_connection_retries"

Reply:
I believe dbt wants it to be `retries`.